### PR TITLE
Janitorial ERT rekit/buff and a few additions for the jani ert squad.

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

Adds a new janitorial toolset implant, that is mostly the same, but with a bluespace light replacer, the big space cleaner spray bottle and deluxe soap, the implant is also EMP proof.
Removes the bluespace light replacer, and a spray bottle form their bags/belt, as they're in the implant now and dont need two, and moves the bluespace trash bag to the belt.
Adds a new MODsuit module, "boot heating" to dry floors as you walk, much like the dry galoshes, for GAMMA jani ert.
Gives amber ert dry galoshes, adds and gives red ert lightweight dry galoshes, as they have heavy armor already, they don't need double slowdown.
Replaces the Red ert mini egun with a normal one, and replaces the Gamma ert egun with a advanced energy gun, for fighting those pesky mice and man eater kudzu, and for self defense. 
Amber ert now get a bluespace trashbag, and light replacer.
## Why It's Good For The Game

Janitorial ERT are largely a jokey send/bait for people to sign up, so bringing them ahead of the stations janitors in the context of tools is something i have wanted to do for a while. 
So this PR brings them quite abit ahead of the uncommon robust/knowledgeable janitors that sometimes come around, as well as some QOL stuff, and less rushing to RND to get basic upgrades in the case of amber ert, and less slowdown in the case of red.

## Testing

Compiled, checked if all my changes and additions functioned correctly, and they did.
## Changelog
:cl:
add: Added a new advanced janitorial implant for ERT janitors 
add: Added a new modsuit module for GAMMA janitors, that allow the modsuit shoes to act as dry galoshes.
add: Added lightweight galoshes for Red janitorial ert.
tweak: Tweaked the contents of the bags and belts of the janitorial ert, they no longer get a light replacer, and they're down a advanced space cleaner, as they're in the implant now, and the BS trashbag is in the belt now.
tweak: Red ert janitors now get a basic energy gun, and GAMMA gets a advanced energy gun.
tweak: Amber janitor ert now get a seclite rather than a flashlight
tweak: GAMMA janitor ert now have a push broom in their bags.
/:cl:

### CODE REVIEW

**Comment by Aquilar** [comment]
add an ERT janitor martial art while holding a push broom. :D

File: code/modules/response_team/ert_outfits.dm [hunk 1]
```diff
@@ -542,15 +541,13 @@
 		/obj/item/grenade/chem_grenade/antiweed = 2,
 		/obj/item/grenade/clusterbuster/cleaner = 1,
 		/obj/item/storage/box/lights/mixed = 1,
-		/obj/item/storage/bag/trash/bluespace = 1,
-		/obj/item/lightreplacer/bluespace = 1,
-		/obj/item/melee/flyswatter = 1,
-		/obj/item/gun/energy/gun = 1
+		/obj/item/push_broom,
```
Comment (Burzah) [inline code/modules/response_team/ert_outfits.dm:544]:
```suggestion
		/obj/item/push_broom = 1,
```

File: code/modules/surgery/organs/augments_arms.dm [hunk 2]
```diff
@@ -323,6 +323,19 @@
 	parent_organ = "l_arm"
 	slot = "l_arm_device"
 
+/obj/item/organ/internal/cyberimp/arm/janitorial/advanced /// ERT implant, i dont overly expect this to get into the hands of crew
+	name = "advanced janitorial toolset implant"
+	desc = "A set of advanced janitorial tools hidden behind a concealed panel on the user's arm"
```
Comment (Burzah) [inline code/modules/surgery/organs/augments_arms.dm:328]:
```suggestion
	desc = "A set of advanced janitorial tools hidden behind a concealed panel on the user's arm."
```

File: code/modules/clothing/shoes/misc_shoes.dm [hunk 3]
```diff
@@ -57,6 +57,12 @@
 	desc = "A pair of purple rubber boots, designed to prevent slipping on wet surfaces while also drying them."
 	icon_state = "galoshes_dry"
 
+/obj/item/clothing/shoes/galoshes/dry/lightweight /// for red janitor ert.
+	name = "Lightweight absorbent galoshes"
```
Comment (DGamerL) [inline code/modules/clothing/shoes/misc_shoes.dm:61]:
```suggestion
	name = "lightweight absorbent galoshes"
```

File: code/modules/clothing/shoes/misc_shoes.dm [hunk 4]
```diff
@@ -57,6 +57,12 @@
...
+	desc = "A pair of expensive looking lightweight rubber boots, designed to prevent slipping on wet surfaces while also drying them."
+	icon_state = "galoshes_dry"
```
Comment (DGamerL) [inline code/modules/clothing/shoes/misc_shoes.dm:63]:
```suggestion
```
It inherits this from the base dry galoshes

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**